OpenOversight/tests/routes/test_officer_and_department.py

```diff
@@ -103,6 +104,30 @@ def test_user_can_access_officer_list(mockdata, client, session):
         assert "Officers" in rv.data.decode("utf-8")
 
 
+@pytest.mark.parametrize(
+    "filter_func, has_placeholder",
+    [
+        (Operators.__invert__, True),
+        (lambda x: x, False),
+    ],
+)
```

Comment:
Sorry I have to ask, but how does this all work? I'm having a hell of a time trying to decifer it all and how it's used in the test 😅

Reply:
Ohhhh I see, each of the tuples contain the values for `filter_func` and `has_placeholder`. I was reading this totally wrong 🤦‍♀️ 

I _think_ I prefer explicitly named tests for cases like this... hopefully it's something a copy/paste job wouldn't make too annoying to write the first time. But it might be harder to maintain. I'll leave it up to you!

Reply:
Yea I'm a huge fan of parametrized tests, especially because the tests are exactly the same beyond these two conditions! I'll add a comment here at least though, should have done that in the first place